README.md

```diff
@@ -6,39 +6,189 @@ Please edit this README.md file with your answers to the worksheet questions.
+|    T    |    F    |    T    |    T                              |
+|    T    |    T    |    F    |    F                              |
+|    T    |    T    |    T    |    F                              |
+
 ### d
 
+|    A    |    B    |    C    |    D    |    A and not (B or not C) and (not A and D)    |
+|---------|---------|---------|---------|------------------------------------------------|
+|    F    |    F    |    F    |    F    |    F                                           |
+|    F    |    F    |    F    |    T    |    F                                           |
+|    F    |    F    |    T    |    F    |    F                                           |
+|    F    |    F    |    T    |    T    |    F                                           |
+|    F    |    T    |    F    |    F    |    F                                           |
+|    F    |    T    |    F    |    T    |    F                                           |
+|    F    |    T    |    T    |    F    |    F                                           |
+|    F    |    T    |    T    |    T    |    F                                           |
+|    T    |    F    |    F    |    F    |    F                                           |
+|    T    |    F    |    F    |    T    |    F                                           |
+|    T    |    F    |    T    |    F    |    F                                           |
+|    T    |    F    |    T    |    T    |    F                                           |
+|    T    |    T    |    F    |    F    |    F                                           |
+|    T    |    T    |    F    |    T    |    F                                           |
+|    T    |    T    |    T    |    F    |    F                                           |
+|    T    |    T    |    T    |    T    |    F                                           |
+
 ## Question 2
 
 ### a
-
+![2A](https://raw.githubusercontent.com/TrainerIsaac/comp110-worksheet-4/master/2A.jpg)
 ### b
-
+![2B](https://raw.githubusercontent.com/TrainerIsaac/comp110-worksheet-4/master/2B.jpg)
 ### c
-
+![2C](https://raw.githubusercontent.com/TrainerIsaac/comp110-worksheet-4/master/2C.jpg)
 ### d
+![2D](https://raw.githubusercontent.com/TrainerIsaac/comp110-worksheet-4/master/2D.jpg)
 
 ## Question 3
 
 ### a
 
+| A | B | Not (A or B) |
+|---|---|--------------|
+| F | F | T            |
+| F | T | F            |
+| T | F | F            |
+| T | T | F            |
+
+=
+
+| a | b | not A and not B |
+|---|---|-----------------|
+| F | F | T               |
+| F | T | F               |
+| T | F | F               |
+| T | T | F               |
+
 ### b
 
+| a | b | not (a and b) |
+|---|---|---------------|
+| F | F | T             |
+| F | T | T             |
+| T | F | T             |
+| T | T | F             |
+
+=
+
+| a | b | a or not b |
+|---|---|------------|
+| F | F | T          |
+| F | T | F          |
+| T | F | T          |
+| T | T | T          |
```

Comment:
The first table is correct, but there are a couple of mistakes in the second one